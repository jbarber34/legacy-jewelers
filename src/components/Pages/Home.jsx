import HomeNav from '../NavBars/HomeNav';
import Title from '../Title';
import Footer from '../Footer';
import Contact from '../Pages/Contact';

function Home() {
  return (
    <div>
      <HomeNav />
      <div className='grid auto-rows-max'>
        {/* Title Section */}
        <div className='container-fluid grid-rows-1 bg-legacyGold'>
          <Title />
        </div>
        {/* Contact Us Section */}
        <div className='lg:my-12'>
          <Contact />
        </div>
        <div className='grid grid-rows-1 bg-black'>
          <Footer />
        </div>
      </div>
    </div>
  );
}

export default Home;
