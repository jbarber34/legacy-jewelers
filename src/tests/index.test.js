import { render, screen, fireEvent } from '@testing-library/react';
import Home from '../pages/index';

//Demo Test

it('renders homepage unchanged', async () => {
  render(<Home />);
  const button = await screen.getByTestId('btn-home');

  fireEvent.click(button);
});
