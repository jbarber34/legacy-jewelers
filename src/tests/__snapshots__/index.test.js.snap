// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="bg-red-400 p-3"
  >
    <button
      class="mantine-Button-filled mantine-Button-root mantine-1grzg0q"
      id="btn-home"
      type="button"
    >
      <div
        class="mantine-3xbgk5 mantine-Button-inner"
      >
        <span
          class="mantine-qo1k2 mantine-Button-label"
        >
          Working 
        </span>
      </div>
    </button>
  </div>
</div>
`;
